raytracing/figure.py

```diff
@@ -12,27 +13,43 @@ class Figure:
     """
     def __init__(self, opticalPath):
         self.path = opticalPath
+        self.raysList = []
 
-        self.graphics = []
-        self.lines = []
+        self.graphicGroups = {'FOV': [], 'object': [], 'lamp': [], 'elements': []}
```

Comment:
FOV should be renamed. We really mean "principal and axial rays from FOV".
Object should be: "Object-Image" I think?